### PR TITLE
tanharpatel

### PR DESCRIPTION
**Pull request by tanharpatel**
- Made app responsive (fixed all overflows)
- Edited result page in regards of displaying answers 
- Homepage made little better
- Fixed the error (previously opening math puzzle no. 3 when clicked 4)

I have read the Code Of Conduct.
I have followed all the steps of submission properly.

### CODE REVIEW

**Comment by tanharpatel** [comment]
@harsh2201 this commit is with latest changes and is able to merge directly without merge conflicts

**Comment by tanharpatel** [comment]
@harshptl14 this commit is with latest changes and is able to merge directly without merge conflicts.

File: lib/homePage.dart [hunk 1]
```diff
@@ -90,17 +90,28 @@ class CategoryCard extends StatelessWidget {
           onTap: onPressed,
           child: Container(
               decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(30),
```
Comment (harshptl14) [inline lib/homePage.dart]: This circular Container looks amazing, but the whole application is based on a rectangular shape. So in this application Ragtengle would be better! what do you think?

File: lib/homePage.dart [hunk 2]
```diff
@@ -90,17 +90,28 @@ class CategoryCard extends StatelessWidget {
...
               height: MediaQuery.of(context).size.width * 0.5,
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.center,
                 mainAxisAlignment: MainAxisAlignment.center,
                 children: [
-                  SvgPicture.asset(
-                    icon,
-                    height: 40.0,
-                    width: 40.0,
+                  ShaderMask(
```
Comment (harshptl14) [inline lib/homePage.dart]: And this color also looks amazing🔥, but the theme of the application is Black and white so I think black and white would look nice! what do you think?

**Review by harshptl14 — COMMENTED** [review]
The other changes were amazing✨🎯, I just think that the previous home screen would match with the theme of the application! What do you think?

**Comment by tanharpatel** [comment]
Ok i will do the previous one :)

**Review by harshptl14 — APPROVED** [review]
Thanks,@tanharpatel, for opening the pull request! 👏

I have reviewed your pull request thoroughly, and it's perfect!

Star ⭐ this project would be great! You can also [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions🤟
Join the [discord channel of club gamma🙌](https://discord.gg/kjnp6wU)

**Pull request merged**